riptide-spring-boot-autoconfigure/src/main/java/org/zalando/riptide/autoconfigure/ValueConstants.java

```diff
@@ -0,0 +1,14 @@
+package org.zalando.riptide.autoconfigure;
+
+/**
+ * Constants refering to beans that need to be looked up by types during Riptide clients constructions.
+ * <p>This is an artificial arrangement of 16 unicode characters, with its sole purpose
+ * being to never match user-declared values.
+ */
+class ValueConstants {
```

Comment:
This deserves a Google Gag remark annotation, e.g. `@Magic`.

Reply:
Added magic, but it seems to me that Hack is more applicable here